nifi-nar-bundles/nifi-spark-bundle/nifi-livy-processors/src/main/java/org/apache/nifi/processors/livy/ExecuteSparkInteractive.java

```diff
@@ -129,7 +185,13 @@
     public void init(final ProcessorInitializationContext context) {
         List<PropertyDescriptor> properties = new ArrayList<>();
         properties.add(LIVY_CONTROLLER_SERVICE);
+        properties.add(IS_BATCH_JOB);
+        properties.add(PY_FILES);
+//        properties.add(JAR_FILES);
+        properties.add(MAIN_PY_FILE);
+        properties.add(NAME);
         properties.add(CODE);
+//        properties.add(ARGS);
```

Comment:
Only `pyfiles` and `file` options are tested. Rest are yet to be tested.

Plan was to go with implementing test modules and test other features, since the current manual testing takes a long routine of compile, copy and restart of the Nifi.